src/components/InputsTable.js

```diff
@@ -0,0 +1,205 @@
+import _ from 'lodash/fp'
+import { useState } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+import { AutoSizer } from 'react-virtualized'
+import { icon } from 'src/components/icons'
+import {
+  InputSourceSelect,
+  ParameterValueTextInput,
+  parseMethodString,
+  RecordLookupSelect,
+  StructBuilderLink
+} from 'src/components/submission-common'
+import { FlexTable, HeaderCell, Sortable, TextCell } from 'src/components/table'
+import TooltipTrigger from 'src/components/TooltipTrigger'
+import colors from 'src/libs/colors'
+import * as Utils from 'src/libs/utils'
+import { StructBuilderModal } from 'src/pages/StructBuilderModal'
+
+
+const buildStructPath = ({ structBuilderPath, headTemplate, pathTemplate, lastTemplate }) => {
```

Comment:
I think to help better understand, it might be worth adding comments about what these 4 functions do, maybe with examples?